src/main/groovy/io/gatling/gradle/GatlingEnterprisePackageTask.groovy

```diff
@@ -93,9 +93,11 @@ class GatlingEnterprisePackageTask extends Jar {
     }
 
     private void collectDepAndChildren(ResolvedDependency dep, Set<ResolvedDependency> acc) {
-        acc.add(dep)
-        for (child in dep.children) {
-            collectDepAndChildren(child, acc)
+        if (!acc.contains(dep)) {
```

Comment:
So cool.
But I'm just wondering how the user end up in this situation, compilers don't like cyclic dependencies in my memories,  I'm curious to know how such a jar is generated.

Reply:
The scope of these dependencies is `runtime`, not `compile`.